support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/SupporterPlusSubcriptionBuilder.scala

```diff
@@ -35,6 +35,11 @@ class SupporterPlusSubcriptionBuilder(
     val todaysDate = dateGenerator.today
 
     val contributionAmount = state.product.amount - getBaseProductPrice(productRatePlanId, state.product.currency)
+    if (contributionAmount < 0)
```

Comment:
Should we block for `<= 0`?

Reply:
No because 0 is expected, it's the normal case where people are just paying the subscription amount

Reply:
Gotcha, it's the contribution amount.